framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -689,6 +693,8 @@ def set_monitors(self, **data):
 
             if cherrypy.request.method != 'POST':
```

Comment:
Don't we have here 6 lines repeated at least 3 times ?

Reply:
Which lines are you thinking about?

In any case, at the end when I have no errors at loading, I will review the code and extract duplicated code in functions.